Formula/pfetch-rs.rb

```diff
@@ -0,0 +1,24 @@
+class PfetchRs < Formula
+  desc "Pretty system information tool written in Rust"
+  homepage "https://github.com/Gobidev/pfetch-rs"
+  url "https://github.com/Gobidev/pfetch-rs/archive/refs/tags/v2.2.0.tar.gz"
+  sha256 "b21a91a9536ca776c38541e3d79a6c8fab4b522db0ef15353cb11e0833e7d3f7"
+  license "MIT"
+  head "https://github.com/Gobidev/pfetch-rs", branch: "main"
```

Comment:
You need a `.git` here:

```suggestion
  head "https://github.com/Gobidev/pfetch-rs.git", branch: "main"
```

Reply:
thank you for the quick reply. I will close this PR and wait for the project to mature. 

Have a great day.